src/server/api/qrcode.ts

```diff
@@ -0,0 +1,24 @@
+import Express from 'express'
+
+import createGoQrCode from '../util/qrcode'
+import ImageFormat from '../../shared/util/imageFormat'
+
+const router = Express.Router()
+
+router.post('/', (req, res) => {
+  const url = req.body.url as string
+  const format = req.body.format as ImageFormat
+  if (!url || !format) {
+    res
+      .status(400)
+      .send('Please include the required "url" and "format" headers')
+    return
```

Comment:
Perhaps some validators can be useful too since this endpoint requires a very specific type of data to be passed in. If you branch off the file-upload-serverside PR branch, you'll get access to Joi's validators which might make it easy

Reply:
Added Joi validators, it should be working pretty fine now